src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -30,11 +30,23 @@ import { fetchPipelineRuns } from '../../actions/pipelineRuns';
 import {
   getPipelineRunsByPipelineName,
   getPipelineRunsErrorMessage,
+  getSelectedNamespace,
   isFetchingPipelineRuns
 } from '../../reducers';
 
 export /* istanbul ignore next */ class PipelineRuns extends Component {
-  async componentDidMount() {
+  componentDidMount() {
+    this.fetchPipelineRuns();
+  }
+
+  componentDidUpdate(prevProps) {
+    const { namespace } = this.props;
+    if (namespace !== prevProps.namespace) {
```

Comment:
We also want to do this if pipelineName has changed

Reply:
Done 👍